src/component/feature/FeatureView2/FeatureStrategies/FeatureStrategiesEnvironments/FeatureStrategiesRefresh/FeatureStrategiesRefresh.tsx

```diff
@@ -0,0 +1,52 @@
+import { Button } from '@material-ui/core';
+import AnimateOnMount from '../../../../../common/AnimateOnMount/AnimateOnMount';
+import { useStyles } from './FeatureStrategiesRefresh.styles';
+
+interface IFeatureStrategiesRefreshProps {
+    show: boolean;
+    refresh: () => void;
+    cancel: () => void;
+}
+
+const FeatureStrategiesRefresh = ({
+    show,
+    refresh,
+    cancel,
+}: IFeatureStrategiesRefreshProps) => {
+    const styles = useStyles();
+
+    return (
+        <AnimateOnMount
+            mounted={show}
+            enter={styles.fadeInEnter}
+            start={styles.fadeInStart}
+            leave={styles.fadeInLeave}
+        >
+            <div className={styles.container}>
+                <p className={styles.refreshHeader}>
+                    NOTE: Updated configuration
+                </p>
+                <p className={styles.paragraph}>
+                    There is new strategy configuration available. This might
+                    mean that someone has updated the strategy configuration
+                    while you were working. Would you like to refresh?
```

Comment:
This is really cool.

Next; we should figure out how to make strategies CRDTs so we can support simultaneous edits :)